UI/src/view/point_asset/pointAssetForm.js

```diff
@@ -75,7 +81,28 @@ root.PointAssetForm = function() {
     });
   };
 
-  this.renderValueElement = function(asset, collection) { return ''; };
+  var propertyOrdering = ['suggest_box'];
```

Comment:
You can only render the suggested box if the asset has the isSuggestedAsset: true. If this variable is false, the renderer knows not to create a suggestion box for the asset in question. The idea here remained the same but now it is more generic instead of calling the render method on each asset we now call a component renderer with the components we want to render. So there isn't much difference between calling the function explicitly or running the method if we have the suggested box as a component. Either way you still have to use it in addition to having the isSuggestedAsset variable.

Reply:
Ok, I understand.